release/models/virtual-devices/openconfig-virtual-device-types.yang

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2022 Intel Corporation.
+// SPDX-License-Identifier: Apache-2.0
+
+module openconfig-virtual-device-types {
+
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/virtual-device-types";
+
+  prefix "oc-vdevice-types";
+
+  import openconfig-extensions { prefix oc-ext; }
+
+  organization "IPDK - Infrastructure Programming Development Kit";
+
+  contact
+    "IPDK Admin
+    ipdkadmin@gmail.com";
```

Comment:
Does this email address actually exist? Who has control of this address?

Reply:
@dandaly can you confirm?

Reply:
Yes this email exists and I monitor it.